src/domain/application/components/summary/JohtoselvitysBasicInformationSummary.tsx

```diff
@@ -78,12 +73,8 @@ const BasicInformationSummary: React.FC<Props> = ({ formData, children }) => {
       <SectionItemContent>
         <p style={{ whiteSpace: 'pre-wrap' }}>{workDescription}</p>
       </SectionItemContent>
-      {!features.accessRights && (
-        <>
-          <SectionItemTitle>{t('form:labels:omatTiedot')}</SectionItemTitle>
-          <SectionItemContent>{orderer && <ContactSummary contact={orderer} />}</SectionItemContent>
-        </>
-      )}
+      <SectionItemTitle>{t('form:labels:omatTiedot')}</SectionItemTitle>
```

Comment:
"Omat tiedot" section can be left out and it's not in Figma either.

Reply:
Thank you for noticing! Fixed it.